mods/ra/mod.yaml

```diff
@@ -187,7 +187,7 @@ Fonts:
 		Font:./mods/common/FreeSansBold.ttf
 		Size:14
 	Title:
-		Font:./mods/ra/ZoodRangmah.ttf
+		Font:ra:ZoodRangmah.ttf
 		Size:48
 	MediumBold:
 		Font:./mods/common/FreeSansBold.ttf
```

Comment:
Same argument as above.
Common (and `all`) is basically a system (engine) thing. The mods are packages that may or may not be there, depending on whether the user wants them.
So I'm not convinced about either requested fixup.

Does everyone disagree with me?